ring_doorbell/__init__.py

```diff
@@ -132,7 +144,6 @@ def _authenticate(self, attempts=RETRY_TOKEN, session=None, wait=1.0):
             loop += 1
 
             try:
-                if session is None:
```

Comment:
The intention here looks wrong now.

Reply:
@d0ugal, it was fixed by the PR #168 

Thanks for the heads up!

Reply:
Ah well! This should not have been there as well, my apologies. Thanks @d0ugal!